frontend/src/components/NutritionFacts.jsx

```diff
@@ -0,0 +1,141 @@
+import ingredients from '../data/ingredients.json'
+import nutritionData from '../data/nutrition_data.json'
+import './NutritionFacts.css';
+
+export default function NutritionFacts(props) {
+
+    let calories = 0;
```

Comment:
If you move the `sumNutrition` inside `sumNutritionData`, you'll also have to move `calories` inside that function as well.

You could either return them as an array 

```js
return [calories, sumNutrition];
```

Or as an object 

```js
return {calories, sumNutrition};
```

And destructure as needed

```js
const [calories, nutritionFacts] = sumNutritionData(propsrecipe);
```